libs/feature/record/src/lib/record-metadata/record-metadata.component.html

```diff
@@ -36,12 +36,20 @@
     </div>
   </mat-tab>
 
-  <mat-tab [disabled]="(displayMap$ | async) === false">
+  <mat-tab
+    [disabled]="
+      (displayMap$ | async) === false && (displayData$ | async) === false
```

Comment:
What is `displayMap$` if it's not about displaying the map ?

Reply:
True, not very meaningful what I wrote